IntelliTect.TestTools.Data/DatabaseFixture.cs

```diff
@@ -15,7 +15,7 @@ public class DatabaseFixture<TDbContext> : IDisposable where TDbContext : DbCont
 
         private Lazy<DbContextOptions<TDbContext>> Options { get; }
 
-        private IServiceProvider? ServiceProvider { get; set; }
+        private IServiceProvider ServiceProvider { get; set; }
```

Comment:
I don't think this is correct. It is not set in the ctor, so it will be null directly after instantiation.